src/test/java/alison/TaskTest.java

```diff
@@ -0,0 +1,18 @@
+package alison;
+
+import alison.exception.AlisonException;
+import alison.task.Deadline;
+import alison.task.Task;
+import alison.tool.Parser;
+import org.junit.jupiter.api.Test;
```

Comment:
Perhaps a line break between lines 6 and 7 would make the import statements look neater? 😄